app/src/main/res/layout-land/ongoing_topic_item.xml

```diff
@@ -57,18 +57,17 @@
 
         <TextView
           android:id="@+id/topic_name_text_view"
+          style="@style/TextViewStart"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
           android:layout_marginStart="8dp"
           android:layout_marginTop="8dp"
           android:layout_marginEnd="8dp"
           android:ellipsize="end"
           android:fontFamily="sans-serif"
-          android:gravity="start"
```

Comment:
TextViewStart only specifies textAlignment; should the gravity property stay or does it also need to be part of the style? Though, I am wondering whether this is even used given that the text view is in a ConstraintLayout.

Reply:
It is not being used so removed it.